pkg/rbac/middleware.go

```diff
@@ -5,15 +5,29 @@ import (
 	"encoding/json"
 	"net/http"
 	"strings"
+	"sync"
+	"time"
 
 	"github.com/stolostron/search-v2-api/pkg/config"
 	authv1 "k8s.io/api/authentication/v1"
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 	"k8s.io/klog/v2"
 )
 
+//SAVING THIS IN ONLY BRANCH ANX/AUTHENTICATE
+
+const THREADING = true
```

Comment:
Why do we need this const? Threading is enabled by default in go and we aren't changing that setting.

Reply:
For mostly testing the lock field, i.e we could change the variable to false but we can probably set in the test file once that test is created instead of putting it here.

Reply:
Yes, let's keep it simple.